src/editor/FileLoader.java

```diff
@@ -0,0 +1,77 @@
+package editor;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+
+public class FileLoader {
+
+    static String loadFile(String fileName)  {
+        File file = new File(fileName);
+        /*
+           If file exists and not empty, read it
+        */
+        if (file.isFile()) {
+            try {
+               return Files.readString(Paths.get(fileName));
+            } catch (IOException e) {
+                e.printStackTrace();
+                return null;
```

Comment:
As a hind, In real programming, it is better to avoid nulls whenever is possible.